clustering/k-means-clustering-plugin/VERSION

```diff
@@ -0,0 +1 @@
+0.3.2
```

Comment:
Why is the version jumping between version 0.2.30 to 0.3.2?

Please review the semantic versioning documentation.

Reply:
I started working using Nina's already opened PR where she bumped version from 0.2.29 -> 0.3.1
https://github.com/PolusAI/polus-plugins/pull/264/files#diff-91078c3c991ac2f1b53d70b3dd7fe466fc35b8e8ff8ecbad23ef6701a47ad52c

Reply:
I only bumped up patch version